llvm/test/Transforms/Inline/PowerPC/inline-target-features.ll

```diff
@@ -0,0 +1,37 @@
+; RUN: opt < %s -mtriple=powerpc64le-unknown-linux-gnu -S -passes=inline | FileCheck %s
```

Comment:
nit: can we use `utils/update_test_checks.py` to automatically generate the check lines? Seems the case is quite simple.